src/Commands/BlockMake.php

```diff
@@ -54,12 +54,15 @@ public function getBlockMaker(): \A17\Twill\Services\Blocks\BlockMaker
      */
     public function handle()
     {
+        $generateView = $this->confirm('Should we also generate a view file for rendering the block?');
```

Comment:
@ifox Thinking since this is 3.x only, should we swap this with an option like `--noPreview` to skip preview generation and do it by default if no option is provided?

Reply:
Let's not do this as decoupled frontends have no use for this.